src/interface.rs

```diff
@@ -293,4 +293,22 @@ impl HWIClient {
             status.into()
         })
     }
+
+    /// Set logging level
+    /// # Arguments
+    /// * `debug` - Print debug statements if true.
+    pub fn set_debug(debug: bool) -> Result<(), Error> {
```

Comment:
Maybe we could instead have a `set_log_level`, so that the users can set their preferred log level (warn, debug, info, error...)